demos/embeddings/gcn-unsupervised-graph-embeddings.ipynb

```diff
@@ -0,0 +1,737 @@
+{
```

Comment:
<blockquote>This demo demonstrated training a graph classification model without supervision. This model could be used to compute embedding vectors or representations for graphs.</blockquote><p>Is it technically more like "unsupervsed representation learning on graphs" + "supervised graph classification", rather than referring to the whole demo as being unsupervised? It looks like a supervised classification workflow towards the bottom if I understood correctly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
This follows our demos for other unsupervised node classification tasks, where representations are computed without supervision and then there's example downstream tasks which may or may not be supervised. For example, unsupervised graphsage: https://stellargraph.readthedocs.io/en/latest/demos/embeddings/graphsage-unsupervised-sampler-embeddings.html#Downstream-task

I'll tweak the heading to hopefully make this clearer, replacing "Evaluate the model" with "Downstream tasks".